### PR TITLE
chore: fix logging issues

### PR DESCRIPTION
**Pull request by Lightning00Blade**
Some variable renaming cleanup

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ name: E2E tests
 permissions: read-all
 
 env:
-  DEBUG: 'bidi:server:info,bidi:mapper:debug:*'
+  DEBUG: 'bidi:server:*,bidi:mapper:debug:*'
```
Comment (Lightning00Blade) [inline .github/workflows/e2e.yml:9]: Decided to go with this rather then update `DEBUG` when `VERBOSE` so they don't depend on each. Else it becomes problematic with mix and matching.

File: src/bidiTab/mapperTabPage.ts [hunk 2]
```diff
@@ -69,8 +69,11 @@ export function log(logType: LogPrefix, ...messages: unknown[]) {
     return;
   }
 
-  // If `sendDebugMessage` is defined, send the log message there.
-  global.window?.sendDebugMessage?.(JSON.stringify({logType, messages}));
+  // Skip sending BiDi logs as they are logged once by `bidi:server:*`
+  if (!logType.startsWith(LogType.bidi)) {
```
Comment (Lightning00Blade) [inline src/bidiTab/mapperTabPage.ts:73]: Move to here as it makes more sense to not send the info back rather then sending and filtering on the other end.

File: src/utils/log.ts [hunk 3]
```diff
@@ -21,6 +21,7 @@ export enum LogType {
   cdp = 'cdp',
   debug = 'debug',
   debugError = 'debug:error',
+  debugInfo = 'debug:info',
```
Comment (Lightning00Blade) [inline src/utils/log.ts:24]: If you set `bidi:debug:*` things with `bidi:debug` prefix will not be shown.

File: tools/run-bidi-server.mjs [hunk 4]
```diff
@@ -60,7 +60,8 @@ if (CHANNEL === 'local') {
 }
 
 // DEBUG = (empty string) is allowed.
-const DEBUG = process.env.DEBUG ?? '*';
+let DEBUG = process.env.DEBUG ?? 'bidi:*';
```
Comment (Lightning00Blade) [inline tools/run-bidi-server.mjs:63]: Don't show every package that uses `debug`

File: src/bidiServer/MapperServer.ts [hunk 5]
```diff
@@ -17,21 +17,30 @@
...
 import {WebSocketTransport} from '../utils/websocketTransport.js';
 
 const debugInternal = debug('bidi:mapper:internal');
 const debugInfo = debug('bidi:mapper:info');
-const debugMapperDebugOthers = debug('bidi:mapper:debug:others');
-const debugMapperDebugPrefix = 'bidi:mapper:debug:';
+const debugOthers = debug('bidi:mapper:debug:others');
+// Memorizes a debug creation
+const loggers = new Map<string, Debugger>();
+const getLogger = (type: LogPrefix) => {
```
Comment (Lightning00Blade) [inline src/bidiServer/MapperServer.ts:32]: This should help with speed imo, as internally it does some checks and setups for each call which I am not sure if calling `debug` memorizes itself.

**Review by thiagowfx — APPROVED** [review]

**Pull request merged**